src/offset/local/mod.rs

```diff
@@ -186,6 +189,102 @@ impl TimeZone for Local {
     }
 }
 
+#[cfg(any(unix, windows))]
```

Comment:
I'd like the three Unix-refactoring commits appear before the ones that add new Windows stuff, can you fix that up?